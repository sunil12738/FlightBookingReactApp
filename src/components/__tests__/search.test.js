import React from 'react';
import Search from '../search.jsx';
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
	const component = renderer.create(<Search />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});