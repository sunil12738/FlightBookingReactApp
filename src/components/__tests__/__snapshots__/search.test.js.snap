// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="padding-10px margin-10px border-1px-solid-grey"
>
  <div>
    <input
      className="button "
      onClick={[Function]}
      type="button"
      value="one way"
    />
    <input
      className="button "
      onClick={[Function]}
      type="button"
      value="return"
    />
  </div>
  <div>
    <input
      className=""
      list="city_names"
      onClick={[Function]}
    />
     
    <br />
    <input
      className=""
      list="city_names"
      onClick={[Function]}
    />
     
    <br />
    <datalist
      id="city_names"
    >
      <option
        value="delhi"
      />
      <option
        value="mumbai"
      />
      <option
        value="bengaluru"
      />
      <option
        value="kolkata"
      />
      <option
        value="kanpur"
      />
    </datalist>
    <input
      className=""
      min="2017-04-16"
      onChange={[Function]}
      onClick={[Function]}
      type="date"
      value="2017-04-16"
    />
     
    <br />
    <input
      className=""
      disabled={true}
      min="2017-04-16"
      onClick={[Function]}
      type="date"
    />
     
    <br />
    <input
      className="button"
      onClick={[Function]}
      type="button"
      value="Search"
    />
  </div>
</div>
`;
